FlowCryptUITests/SignInImapTest.swift

```diff
@@ -100,28 +193,52 @@ extension SignInImapTest {
         logger.logInfo("Try to connect")
         wait(10)
     }
+    
+    private func verifyFlowWithNoBackups(for user: UserCredentials) {
+        loginWithImap(user)
+        
+        let tablesQuery = app.tables
+        
+        let noBackupsLabel = tablesQuery.staticTexts["No backups found on account: \n\(user.email)"]
+        let importMyKeyButton = tablesQuery.buttons["Import my key"]
+        let createNewKeyButton = tablesQuery.buttons["Create a new key"]
+        
+        XCTAssert(noBackupsLabel.exists)
+        XCTAssert(importMyKeyButton.exists)
+        XCTAssert(createNewKeyButton.exists)
+        XCTAssert(setupUseAnotherAccount.exists)
+        
+        importMyKeyButton.tap()
+        navigationBackButton.tap()
+        
+        createNewKeyButton.tap()
+        navigationBackButton.tap()
+        
+        importMyKeyButton.tap()
+        
+        let loadFromFileButton = tablesQuery.buttons["Load From File"]
+        XCTAssert(loadFromFileButton.exists)
```

Comment:
We should have a test where we import a key from file, proceed with setup, and decrypt an encrypted message that was in inbox encrypted for that key.